### PR TITLE
added faq component

### PR DESCRIPTION
**Pull request by geraldngjx**
This PR introduces several enhancements and improvements to the Contact Us page. The changes made include:

- Created a header for the Contact Us page to provide a visual representation of the page.
- Added a SectionHeader component to display a prominent title and optional subtitle for the page. The subtitle is shown only in desktop view for improved readability.
- Implemented an Accordion component to organize and display frequently asked questions. Users can expand and collapse individual questions to view their corresponding answers.
- Ensured mobile responsiveness for the Contact Us page to provide an optimal viewing experience on different devices.
- Conducted comprehensive testing to validate the functionality and responsiveness of the page across various screen sizes and devices.

These changes aim to improve the user experience on the Contact Us page by offering a clear visual hierarchy, easy navigation through FAQs, and a consistent experience across different devices.

### CODE REVIEW

File: components/contact/faq.tsx [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+              onClick={() => handleTabClick('General')}
+            >
+              <span className="tab-label">
+                {activeTab === 'General' ? (
+                  <strong className="text-3xl text-[#036167]">General</strong>
+                ) : (
+                  <span className="text-3xl text-[#004F4E]">General</span>
+                )}
+              </span>
+            </div>
```
Comment (JustinPeng13) [inline components/contact/faq.tsx:38]: Could we either make the category font size or column width responsive? Currently, the words are broken up for 768px - 1125px (Software Development strong).
Screenshot 2023-07-12 at 1 52 28 PM

**Review by JustinPeng13 — CHANGES_REQUESTED** [review]
Good work! Looks smooth, with just a few aesthetic changes to be made. Also, `yarn build` seems to be failing. Could you try to resolve it?

File: components/contact/faq.tsx (hunk 1 above)
Comment (geraldngjx) [reply components/contact/faq.tsx]: Hi! Sorry for the delay! Changes have been made.

**Review by JustinPeng13 — APPROVED** [review]
LGTM

**Pull request merged**